cx.ring.Ring.json

```diff
@@ -0,0 +1,142 @@
+{
+    "app-id": "cx.ring.Ring",
```

Comment:
Changed from cx.ring.RingGnome due to the fact that all interfaces and services are registered to  cx.ring.Ring.